app/controllers/branches_controller.rb

```diff
@@ -28,15 +29,17 @@ def edit
 
 	def update
 		if @branch.update(branch_params)
-			redirect_to @branch, notice: "Successfully updated branch"
+			redirect_to @branch
+			flash[:notice] = "Successfully updated branch"
```

Comment:
re:

```ruby
# before
redirect_to @branch, notice: "Successfully updated branch"
```

```ruby
# after
redirect_to @branch
flash[:notice] = "Successfully updated branch"
```
Are all of these `flash[:some_level] = xyz` refactorings for personal preference (i.e. readability), or was there an issue you encountered?